public/content/sv/welcome/install-d-locally.md

```diff
@@ -0,0 +1,43 @@
+# Installera D lokalt
+
+På D språkets webbsida [dlang.org](https://dlang.org) kan den senaste kompilerar
+versionen av referens kompilatorn **DMD** (Digital Mars D)
+[laddas ned](http://dlang.org/download.html) och installeras:
+
+### Windows
+
+* [Installerare](http://downloads.dlang.org/releases/2.x/2.071.1/dmd-2.071.1.exe)
+* eller: [Arkiv](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.windows.7z)
+* genom [chocolatey](https://chocolatey.org/packages/dmd): `choco install dmd`
+
+### Mac OS X
+
+* `.dmg` [fil](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.dmg)
+* eller: [Arkiv](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.osx.tar.xz)
```

Comment:
I just realized that we are hard-coding the latest release here, that will be fixed very soon (#449).
So how about applying `s/2.071.0/{{latest-release}}/g` with your favorite regex tool (e.g. `sed s/2.071.0/{{latest-release}}/g  -i <file> `)?